### DISCUSSION TITLE
QLMarkdown Icon - Proposal

### DISCUSSION THREAD

**Discussion by setanarut** [sbarex/QLMarkdown#37] [Show and tell]
## You can make icns from this png if you like

![QLMarkdown](https://user-images.githubusercontent.com/36481442/103432698-ca1e0300-4bf4-11eb-849f-b30e37b13cb8.png)

## You can use this transparent logo for README.md of Github and other web pages.

![Artboard1_2](https://user-images.githubusercontent.com/36481442/103432965-c3de5580-4bf9-11eb-9a1f-95752be2e5d5.png)

## SVG
https://gist.github.com/hazarek/946ceb9e986d0469b3f4814fccdfb79b

logotype font [Railway Extra-bold 800](https://fonts.google.com/specimen/Raleway?preview.text=QLMarkdown&preview.text_type=custom&category=Sans+Serif,Display&preview.size=36&sort=popularity&width=8&sidebar.open=true&selection.family=Raleway:wght@800#standard-styles)

**Comment by sbarex** [comment]
Many thanks for your work. The new version just released uses this icon!
